.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Push to BGA
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow at 2:22 am every morning, Github says that scheduled actions can be delayed if placed directly on the hour
+  # so I moved it to 22 minutes past the hour
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '22 2 * * *'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
```

Comment:
`build` probably isn't the most accurate job name in this case, since it's more of a deploy, but I find myself not caring much. :-P